dev/com.ibm.ws.anno/test/com/ibm/ws/anno/test/cases/SecFVT_Servlet30_Inherit_war_Test.java

```diff
@@ -92,7 +90,7 @@ public void testSecFVT_Servlet30_Common_war_detail_direct() throws Exception {
                     new PrintWriter(System.out, true));
     }
 
-    public static final String SERVLET_SECURITY_CLASS_NAME = ServletSecurity.class.getName();
+    public static final String SERVLET_SECURITY_CLASS_NAME = "com.ibm.ws.anno.test.cases.ServletSecurity";
```

Comment:
```suggestion
    public static final String SERVLET_SECURITY_CLASS_NAME = "javax.servlet.annotation.ServletSecurity";
```
Shouldn't it be that?